arb_os/codeSegment.mini

```diff
@@ -516,8 +517,8 @@ public impure func translateEvmCodeSegment(
                     if (opcode == 0xfd) { // REVERT
                         return pushEvmInsnCall(opWrapper2Uint(evmOp_revert), restOfCode, evmJumpTable, pcTable);
```

Comment:
Should this also be replaced with `evmOp_revert_knownPc`?

Reply:
Fixed.